src/Controller/DefaultController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class DefaultController extends AbstractController
+{
+    /**
+     * @Route("/", name="home")
+     */
+    public function home()
+    {
+        return $this->render('default/index.html.twig', [
+            'controller_name' => 'DefaultController',
```

Comment:
Update this to allow for a customised homepage instead of using the default template.